_includes/card-problem-messages.md

```diff
@@ -24,7 +24,7 @@ All contract types will have the following URI in front of type:
 ### Acquirer and 3-D Secure Problem Types
 
 All acquirer error types will have the following URI in front of type:
-`https://api.payex.com/psp/errordetail/creditcard/<error-type>`
+`https://api.payex.com/psp/errordetail/<error-type>`
```

Comment:
Since Acquirer and 3-D Secure problems are related to credit card, I'm pretty sure their `type` contains `creditcard`.


```suggestion
`https://api.payex.com/psp/errordetail/creditcard/<error-type>`
```